webapp/src/debugger.tsx

```diff
@@ -139,8 +174,14 @@ export class DebuggerVariables extends data.Component<DebuggerVariablesProps, De
 
         return Object.keys(variables).length == 0 ? <div /> :
             <div className={`ui segment debugvariables ${frozen ? "frozen" : ""}`}>
-                <div className="ui middle aligned list">
-                    {this.renderVariables(variables)}
+                <div className="ui collapsing celled table">
+                <thead>
+                    <tr><th>Variable</th>
```

Comment:
Nitpick: Don't split tags like this, either put them all on the same line or give the open and close tags their own line.